setup.py

```diff
@@ -1,4 +1,6 @@
-from setuptools import setup, find_packages
+import pathlib
```

Comment:
Unused import

Reply:
Merging error. Updated the setup file to use markdown instead of text. Black Formatted and Linted the setup.py file in the latest commit.